Alcatraz/Installers/ATZInstaller.h

```diff
@@ -39,10 +39,10 @@ static NSString *const UPDATING_FORMAT = @"Updating %@...";
                                            completion:(void(^)(NSError *error))completion;
 - (void)updatePackage:(ATZPackage *)package progress:(void(^)(NSString *progressMessage, CGFloat progress))progress
                                           completion:(void(^)(NSError *error))completion;
-- (void)removePackage:(ATZPackage *)package
-           completion:(void(^)(NSError *error))completion;
-
+- (void)removePackage:(ATZPackage *)package completion:(void(^)(NSError *error))completion;
+- (void)whitelistPackage:(ATZPackage *)package completion:(void(^)(NSError *error))completion;
 
+- (BOOL)isPackageBlacklisted:(ATZPackage *)package;
```

Comment:
Not sure how i feel about adding the blacklist thing on the package installer.
Might be worth making it's own class